package.json

```diff
@@ -26,87 +36,80 @@
-    "@babel/preset-react": "^7.12.5",
-    "@babel/preset-typescript": "^7.13.0",
-    "@storybook/addon-essentials": "^7.5.3",
-    "@storybook/cli": "^7.5.3",
-    "@storybook/components": "^7.5.3",
-    "@storybook/manager-api": "^7.5.3",
-    "@storybook/react-webpack5": "^7.5.3",
-    "@storybook/test-runner": "^0.15.1",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.3.0",
-    "@types/node": "^20.9.1",
-    "@types/semver": "^7",
-    "@types/styled-components": "^5.1.26",
-    "auto": "^10.3.0",
-    "babel-loader": "^8.1.0",
-    "boxen": "^5.0.1",
-    "concurrently": "^6.2.0",
-    "dedent": "^0.7.0",
-    "detect-package-manager": "^3.0.1",
-    "jest": "^29.0.1",
-    "jest-environment-jsdom": "^29.0.1",
-    "prettier": "^2.3.1",
-    "prop-types": "^15.7.2",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "rimraf": "^3.0.2",
-    "semver": "^7.5.4",
-    "storybook": "^7.5.3",
-    "styled-components": "^6.1.0",
-    "typescript": "^5.2.2",
-    "zx": "^1.14.1"
+    "@storybook/addon-docs": "next",
+    "@storybook/addon-essentials": "next",
+    "@storybook/addon-interactions": "next",
+    "@storybook/addon-links": "next",
+    "@storybook/blocks": "next",
+    "@storybook/icons": "^1.4.0",
+    "@storybook/react": "next",
+    "@storybook/react-vite": "next",
+    "@storybook/test": "next",
+    "@storybook/theming": "next",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^16.3.0",
+    "@types/node": "^22.15.18",
+    "@types/react": "^19.1.4",
+    "@types/react-dom": "^19.1.5",
+    "@types/semver": "^7.7.0",
+    "@types/styled-components": "^5.1.34",
+    "@vitejs/plugin-react": "^4.4.1",
+    "auto": "^11.3.0",
+    "boxen": "^8.0.1",
+    "dedent": "^1.6.0",
+    "jsdom": "^26.1.0",
+    "npm-run-all2": "^8.0.1",
+    "prettier": "^3.5.3",
+    "prompts": "^2.4.2",
+    "react": "^19.1.0",
+    "react-dom": "^19.1.0",
+    "react-error-boundary": "^6.0.0",
+    "storybook": "next",
+    "tsup": "^8.4.0",
+    "typescript": "^5.8.3",
+    "vite": "^6.3.5",
+    "vitest": "^3.1.3",
+    "zx": "^8.5.4"
   },
   "peerDependencies": {
-    "@storybook/addons": "*",
-    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
-    "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0"
-  },
-  "peerDependenciesMeta": {
-    "react": {
-      "optional": true
-    },
-    "react-dom": {
-      "optional": true
-    }
+    "@storybook/icons": "^1.4.0",
+    "storybook": "^9.0.0"
   },
   "publishConfig": {
     "access": "public",
+    "provenance": true,
     "registry": "https://registry.npmjs.org"
   },
+  "bundler": {
+    "exportEntries": [
+      "src/index.ts"
+    ],
+    "managerEntries": [
+      "src/manager.ts"
+    ],
+    "previewEntries": [
+      "src/preview.ts"
+    ]
+  },
   "storybook": {
     "displayName": "Source Link",
     "supportedFrameworks": [
-      "react"
+      "supported-frameworks"
```

Comment:
Apparently your code is framework-agnostic so it should not be advertised only for React.